common/vm_get_vm_info.yml

```diff
@@ -24,6 +24,9 @@
     vm_hardware_version_num: "{{ vm_config.config.version.split('-')[-1] }}"
     vm_primary_nic_mac: ""
 
+- name: "Wait for mac address is available"
+  include_tasks: vm_wait_primary_nic_mac.yml
+
 - name: "Get VM's network adapter"
   include_tasks: vm_get_network_facts.yml
```

Comment:
I think `vm_get_network_facts.yml` can be removed here as if `vm_wait_primary_nic_mac.yml` passes, you can directly set vm_primary_nic_mac with `gather_network_facts.network_data['0'].mac_addr`.

Reply:
change it. Thanks